### DISCUSSION TITLE
Issue related with VENT_ID when MPI calculation

### DISCUSSION THREAD

**Discussion by kimu-5834** [firemodels/fds#12922] [Q&A] [answered]
Dear developers,

I attempted to test my FDS simulation case, which involves two separate domains divided by a plate in the middle. The setup includes multiple vents to facilitate the diffusion of CO2 from the upper region to the lower region. However, I encountered an error stating that the VENT_ID is not properly set up. When I run the simulation without MPI region decomposition, it works well. 
Interestingly, the error only occurs when decomposing the MPI regions in the vertical direction. I would appreciate it if you could let me know the reason.

 Starting FDS ...

 MPI Process      0 started on b01-4206c

 Reading FDS input file ...

ERROR: Cannot find VENT_ID: 開口部01 for Ductnode: 開口部01 (CHID: base2_15_Z10MPI)

ERROR: FDS was improperly set-up - FDS stopped (CHID: base2_15_Z10MPI)

[base2_15_Z10MPI.zip](https://github.com/firemodels/fds/files/15292692/base2_15_Z10MPI.zip)

**Comment by Er9y714** [comment, marked as answer]
Vents do not align with grids. 

![image](https://github.com/firemodels/fds/assets/26166281/e3181fb4-354a-4a20-b44c-7f5bc4e335c8)

**Comment by mcgratta** [comment]
There are 61000 lines in the input file, and you have labeled everything in Chinese. If you want help, you need to reduce the input file until there is only the one vent that is causing problems. Otherwise, it would take days to sort this out.
